.github/workflows/test-build.yml

```diff
@@ -237,10 +253,20 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.6
+    - name: Update apt-get
+      continue-on-error: true
+      run: sudo apt update
+    - name: Update apt-get
+      continue-on-error: true
+      run: sudo apt-get update
+    - name: Update apt-get
```

Comment:
The lines below here appear to be duplicating the lines above. Is this intentional?

Reply:
Yeah. I caught that. Fixed in the recent commit...